modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -178,7 +178,77 @@ const Example = styled('div')(
 
 ### Responsive Styles
 
-> Coming Soon (https://github.com/Workday/canvas-kit/pull/1779)
+The following Canvas Kit responsive utilities will be released as part of v8:
+
+- **ResponsiveContextProvider**: This is a React Context Provider that will take a width and an
+  optional theme prop and allow components to subscribe to context changes via the
+  `useResponsiveContext` hook.
+
+- **useResponsiveContext Hook**: This hook subscribes to the `ResponsiveContextProvider` and
+  provides helpful utilities to determine whether the current width is within a particular theme
+  breakpoint (`Small: 320px, Medium: 768px, Large: 1024px, Extra Large: 1440px`). Developers can use
+  these utilities when creating container-based responsive styles.
+
+- **useResponsiveContainerStyles**: A hook that allows developers to create container-based
+  responsive styles using style objects. These are net-new utilities that developers can opt-in to
+  start using, but no action is required to uptake this change.
```

Comment:
The net-new bit doesn't need to be here in the Upgrade Guide.
```suggestion
- **useResponsiveContainerStyles**: A hook that allows developers to create container-based
  responsive styles using style objects.
```

Reply:
Updated